packages/web/src/pages/upload-page/components/FinishPageNew.tsx

```diff
@@ -75,13 +82,14 @@ const UploadTrackItem = (props: UploadTrackItemProps) => {
     track,
     trackProgress,
     displayIndex = false,
-    displayArtwork = false
+    displayArtwork = false,
+    ...otherProps
   } = props
   // @ts-ignore - Artwork exists on track metadata object
   const artworkUrl = track.metadata.artwork.url
 
   return (
-    <div className={styles.uploadTrackItem}>
+    <div className={styles.uploadTrackItem} {...otherProps}>
```

Comment:
What are we passing through here?
If we're expecting something, we should add `& HTMLProps<HTMLDivElement>` to this component's props type

Reply:
just passing the key prop through